### ISSUE TITLE
WCAG 1.1.1: Images must have alternate text (img[width="\32 10"][height="\31 20"])

### ISSUE THREAD

**Issue by AdnoC** [AdnoC/Dotfiles#2]
Title: WCAG 1.1.1: Images must have alternate text (img[width="\32 10"][height="\31 20"])
Tags: Accessibility, WCAG 1.1.1, image-alt

Issue: Images must have alternate text (image-alt: https://dequeuniversity.com/rules/axe/3.1/image-alt?application=axeAPI)

Target application title: Mars Commuter: Travel to Mars for Work or Pleasure!
Target application url: https://dequeuniversity.com/demo/mars/

Element path: li:nth-child(15) > a[href="mars2.html?a=crater_adventure"] > img[width="\32 10"][height="\31 20"]

Snippet: `<img src="/assets/demo-sites/mars/images/mars-spaceman.jpg" class="" width="210" height="120">`

How to fix:
Fix any of the following:
  Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute or the title attribute is empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"

Environment:
Chrome version 71.0.3578.98

====

This accessibility issue was found using Accessibility Insights for Web, a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
